pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -331,7 +331,7 @@ func (t *Tenant) KESServiceEndpoint() string {
 		Scheme: scheme,
 		Host:   t.KESServiceHost(),
```

Comment:
Its better to add it here @kerneltime 

```suggestion
		Host:   net.JoinHostPort(t.KESServiceHost(), strconv.Itoa(KESPort)),
```

Because url.String() has potential to add `/` at the end which can become like `https://<hostname>/:port`

Reply:
I added a unit test to make sure it is inline with expectation..

Reply:
Yeah but it is safe to avoid that since its natural for url.String() to add a "/" its better to be added at Host: instead of appended to a string. 

net.JoinHostPort() is the correct way to combine host and port